cmd/snap/cmd_snap_op.go

```diff
@@ -744,7 +744,10 @@ func (x *cmdRefresh) refreshMany(snaps []string, opts *client.SnapOptions) error
 		return showDone(x.client, chg, refreshed, "refresh", opts, x.getEscapes())
 	}
 
-	fmt.Fprintln(Stderr, i18n.G("All snaps up to date."))
+	_, timestamp := x.client.WarningsSummary()
+	if last, _ := lastWarningTimestamp(); !timestamp.After(last) {
```

Comment:
Minor nitpick, but I think renaming `lastWarningTimestamp` to `lastSeenWarningTimestamp` could be less confusing. I know it was already like this, please feel free to ignore this comment if doesn't make sense.